### DISCUSSION TITLE
Best place to ask turtle programing questions?

### DISCUSSION THREAD

**Discussion by Skijet1988** [cc-tweaked/CC-Tweaked#1217] [General]
Where is the best place to ask turtle programing questions?

I don't want to clutter up this forum unless this is the appropriate place to ask them?

**Comment by Lupus590** [comment]
You can try the [(new) forums](https://forums.computercraft.cc/index.php) or Discord. [The Discord](https://discord.computercraft.cc/) is more active so I would go there.

(In case you are wondering why I said new forums, we do have [old forums](http://www.computercraft.info/forums2/) but people can't sign up anymore.)
